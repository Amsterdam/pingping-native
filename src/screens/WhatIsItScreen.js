import React from 'react';
import {View, StyleSheet, Dimensions} from 'react-native';
import SimpleHeader from '../components/Header/SimpleHeader';
import {Content, Container, Text} from 'native-base';

const styles = StyleSheet.create({});

const WhatIsItScreen = props => {
  return (
    <Container>
      <SimpleHeader />
      <Content>
        <Text>WhatIsItScreen</Text>
      </Content>
    </Container>
  );
};

export default WhatIsItScreen;
