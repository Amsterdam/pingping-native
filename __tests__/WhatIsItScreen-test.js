import React from 'react';
import WhatIsItScreen from '../src/screens/WhatIsItScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<WhatIsItScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
