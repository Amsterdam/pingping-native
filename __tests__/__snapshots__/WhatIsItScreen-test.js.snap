// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <View>
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Object {
          "backgroundColor": "#0d2036",
        }
      }
    >
      <View
        noShadow={true}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderBottomColor": "transparent",
              "borderBottomWidth": 0.5,
              "elevation": 0,
              "flexDirection": "row",
              "height": 64,
              "justifyContent": "center",
              "left": 0,
              "paddingLeft": 6,
              "paddingRight": 10,
              "paddingTop": 18,
              "right": 0,
              "shadowColor": null,
              "shadowOffset": null,
              "shadowOpacity": null,
              "shadowRadius": null,
              "top": 0,
            },
            Object {
              "alignItems": "flex-start",
              "backgroundColor": "#0d2036",
              "justifyContent": "flex-start",
            },
          ]
        }
        transparent={true}
      >
        <View
          accessible={true}
          focusable={true}
          forwardedRef={[Function]}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "backgroundColor": "transparent",
              "borderBottomWidth": null,
              "borderColor": "#007aff",
              "borderLeftWidth": null,
              "borderRadius": 5,
              "borderRightWidth": null,
              "borderTopWidth": null,
              "borderWidth": undefined,
              "elevation": 0,
              "flexDirection": "row",
              "height": 45,
              "justifyContent": "center",
              "opacity": 1,
              "paddingBottom": 6,
              "paddingLeft": 6,
              "paddingRight": 6,
              "paddingTop": 6,
              "shadowColor": null,
              "shadowOffset": null,
              "shadowOpacity": null,
              "shadowRadius": null,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 12,
                },
                Array [
                  Object {
                    "color": "#000",
                    "fontSize": 30,
                  },
                  Object {
                    "color": "#007aff",
                    "fontSize": 24,
                    "marginLeft": 16,
                    "marginRight": 16,
                    "paddingTop": 2,
                  },
                  Object {
                    "color": "#fff",
                  },
                ],
                Object {
                  "fontFamily": "anticon",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
            type="AntDesign"
          >
            
          </Text>
        </View>
      </View>
    </RCTSafeAreaView>
  </View>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      automaticallyAdjustContentInsets={false}
      contentContainerStyle={
        Array [
          Object {
            "padding": undefined,
          },
          Object {
            "alignItems": "center",
            "flex": 1,
          },
        ]
      }
      contentInset={
        Object {
          "bottom": 0,
        }
      }
      enableAutomaticScroll={true}
      enableOnAndroid={false}
      enableResetScrollToCoords={true}
      extraHeight={75}
      extraScrollHeight={0}
      getScrollResponder={[Function]}
      handleOnScroll={[Function]}
      keyboardDismissMode="interactive"
      keyboardOpeningTime={250}
      keyboardShouldPersistTaps="handled"
      keyboardSpace={0}
      onScroll={[Function]}
      resetKeyboardSpace={[Function]}
      resetScrollToCoords={
        Object {
          "x": 0,
          "y": 0,
        }
      }
      scrollEventThrottle={1}
      scrollForExtraHeightOnAndroid={[Function]}
      scrollIntoView={[Function]}
      scrollToEnd={[Function]}
      scrollToFocusedInput={[Function]}
      scrollToPosition={[Function]}
      showsVerticalScrollIndicator={true}
      style={
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        }
      }
      update={[Function]}
      viewIsInsideTabBar={false}
    >
      <View>
        <RCTYouTube
          loopProp={true}
          onChangeFullscreen={[Function]}
          onChangeQuality={[Function]}
          onChangeState={[Function]}
          onError={[Function]}
          onProgress={[Function]}
          onReady={[Function]}
          play={false}
          playerParams={
            Object {
              "playerVars": Object {
                "controls": undefined,
                "fs": undefined,
                "loop": 1,
                "modestbranding": undefined,
                "origin": undefined,
                "playlist": "ZG0vCfivpQ8",
                "playsinline": 1,
                "rel": undefined,
                "showinfo": undefined,
              },
              "playlistId": undefined,
              "videoId": "ZG0vCfivpQ8",
            }
          }
          style={
            Array [
              Object {
                "overflow": "hidden",
              },
              Object {
                "alignSelf": "stretch",
                "height": 200,
              },
            ]
          }
          videoId="ZG0vCfivpQ8"
        />
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 1,
              "justifyContent": "space-between",
              "padding": 30,
              "position": "relative",
              "width": 750,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                },
                Object {
                  "fontFamily": "Heavitas",
                  "fontSize": 30,
                  "textAlign": "center",
                },
              ]
            }
            uppercase={false}
          >
            Wat is 
            

            PING PING ?
          </Text>
          <Text
            style={
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              }
            }
            styles={
              Object {
                "textAlign": "center",
              }
            }
            uppercase={false}
          >
            🤷‍♂🤷‍♀
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                },
                Object {
                  "fontFamily": "Raleway",
                  "fontSize": 15,
                  "lineHeight": 25,
                  "textAlign": "center",
                },
              ]
            }
            uppercase={false}
          >
            Wij maken een persoonlijk routeplan die je gaat helpen je (financiële) basis op orde te hebben.
          </Text>
          <Text
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                },
                Object {
                  "fontFamily": "Raleway",
                  "fontSize": 15,
                  "lineHeight": 25,
                  "textAlign": "center",
                },
              ]
            }
            uppercase={false}
          >
            Dan hoef je daar geen zorgen meer over te maken en kan je leuke dingen doen.
          </Text>
          <View
            accessible={true}
            focusable={true}
            forwardedRef={[Function]}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "stretch",
                "backgroundColor": "#fb9f00",
                "borderBottomWidth": null,
                "borderColor": false,
                "borderLeftWidth": null,
                "borderRadius": 57,
                "borderRightWidth": null,
                "borderTopWidth": null,
                "borderWidth": undefined,
                "elevation": 5,
                "flexDirection": "row",
                "height": 45,
                "justifyContent": "center",
                "marginBottom": 5,
                "opacity": 1,
                "paddingBottom": 6,
                "paddingTop": 6,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.25,
                "shadowRadius": 3.84,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "color": "#fff",
                    "fontFamily": "System",
                    "fontSize": 16.5,
                    "marginLeft": 0,
                    "marginRight": 0,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                  },
                  Object {
                    "color": "#fff",
                    "fontFamily": "Heavitas",
                    "fontSize": 14,
                  },
                ]
              }
              uppercase={false}
            >
              Fiks je eigen route
            </Text>
          </View>
          <View>
            <View
              accessible={true}
              focusable={true}
              forwardedRef={[Function]}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "stretch",
                  "backgroundColor": "transparent",
                  "borderBottomWidth": null,
                  "borderColor": "#007aff",
                  "borderLeftWidth": null,
                  "borderRadius": 57,
                  "borderRightWidth": null,
                  "borderTopWidth": null,
                  "borderWidth": undefined,
                  "elevation": 0,
                  "flexDirection": "row",
                  "height": 45,
                  "justifyContent": "center",
                  "marginBottom": 5,
                  "opacity": 1,
                  "paddingBottom": 0,
                  "paddingLeft": 0,
                  "paddingRight": 0,
                  "paddingTop": 0,
                  "shadowColor": null,
                  "shadowOffset": null,
                  "shadowOpacity": null,
                  "shadowRadius": null,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "color": "#007aff",
                      "fontFamily": "System",
                      "fontSize": 16.5,
                      "marginLeft": 0,
                      "marginRight": 0,
                      "paddingLeft": 16,
                      "paddingRight": 16,
                    },
                    Object {
                      "color": "#000",
                      "fontSize": 12,
                      "textAlign": "center",
                    },
                  ]
                }
                uppercase={false}
              >
                Start zonder eigen route
              </Text>
            </View>
            <View
              accessible={true}
              focusable={true}
              forwardedRef={[Function]}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "stretch",
                  "backgroundColor": "transparent",
                  "borderBottomWidth": null,
                  "borderColor": "#007aff",
                  "borderLeftWidth": null,
                  "borderRadius": 57,
                  "borderRightWidth": null,
                  "borderTopWidth": null,
                  "borderWidth": undefined,
                  "elevation": 0,
                  "flexDirection": "row",
                  "height": 45,
                  "justifyContent": "center",
                  "marginBottom": 5,
                  "opacity": 1,
                  "paddingBottom": 0,
                  "paddingLeft": 0,
                  "paddingRight": 0,
                  "paddingTop": 0,
                  "shadowColor": null,
                  "shadowOffset": null,
                  "shadowOpacity": null,
                  "shadowRadius": null,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "color": "#007aff",
                      "fontFamily": "System",
                      "fontSize": 16.5,
                      "marginLeft": 0,
                      "marginRight": 0,
                      "paddingLeft": 16,
                      "paddingRight": 16,
                    },
                    Object {
                      "color": "#000",
                      "fontSize": 12,
                      "textAlign": "center",
                    },
                  ]
                }
                uppercase={false}
              >
                Heb je al een route? Importeer gegevens
              </Text>
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>
</View>
`;
